test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -156,6 +157,31 @@ public void AutoFitRankingTest()
                 Assert.True(col.Name == expectedOutputNames[col.Index]);
         }
 
+        [LightGBMFact]
+        public void AutoFitRankingCVTest()
```

Comment:
This is the way experiments are used within codegen. Review: Should I add cross validation tests to all other experiments?

Reply:
It's the other way around. If it has less than 15000 it runs train test split automatically on one of the Execute overloads.

---
In reply to: [447156630](https://github.com/dotnet/machinelearning/pull/5263#discussion_r447156630) [](ancestors = 447156630)

Reply:
I have added CV testing for ranking only. I think it would be good to add testing for other task as well in the future.